projects/lib/src/humanplayer.cpp

```diff
@@ -28,7 +28,8 @@ HumanPlayer::HumanPlayer(QObject* parent)
 
 bool HumanPlayer::canPlayAfterTimeout() const
 {
-	return true;
```

Comment:
So far we've managed to keep all `QSettings` stuff in GUI. In order to maintain this separation, I think we need a method in `ChessPlayer` for setting the `canPlayAfterTimeout` value and the GUI should call that method.